package.json

```diff
@@ -122,12 +122,12 @@
     "react-dropzone": "^14.2.3",
     "react-fast-compare": "^3.2.2",
     "react-image-gallery": "1.2.12",
-    "react-markdown": "^8.0.7",
+    "react-markdown": "^9.0.3",
```

Comment:
Shouldn't we also update:
- `unist-builder`
- `unist-util-visit`
- `hast-util-find-and-replace`

Reply:
I have not downgraded them previously with react-markdown downgrade. But I see that `unist-builder` has a new version so I have installed it here: https://github.com/GetStream/stream-chat-react/pull/2643/commits/611555fc89081a92f29f6ffd7fc40a813449ae32